tests/console/yast_scc.pm

```diff
@@ -16,19 +16,33 @@
 # G-Summary: harmorize zypper_ref between SLE and openSUSE
 # G-Maintainer: Max Lin <mlin@suse.com>
 
-use strict;
 use base "consoletest";
+use strict;
+use warnings;
+use utils;
 
 use testapi;
-use registration;
 
 sub run {
     select_console 'root-console';
 
     if (my $u = get_var('SCC_URL')) {
         type_string "echo 'url: $u' > /etc/SUSEConnect\n";
     }
-    yast_scc_registration;
+    script_run('yast2 scc');
```

Comment:
Better use the same approach as for other test modules. Take a look into tests/console/yast2_* . I think it should be `script_run '…, 0` with the exit code evaluated with `wait_serial`